schema/vrs.yaml

```diff
@@ -249,62 +276,87 @@ definitions:
       always represented by contiguous spans using interbase
       coordinates.
-
-      This definition of SequenceInterval is intended to be compatible with that
-      in Sequence Ontology
-      ([SO:0000001](http://www.sequenceontology.org/browser/current_svn/term/SO:0000001)),
-      with the exception that the GA4GH VRS SequenceInterval may be
-      zero-width. The SO definition is for an "extent greater than
-      zero".
-    additionalProperties: false
     type: object
+    additionalProperties: false
     properties:
       type:
         type: string
-        enum: ["SimpleInterval"]
-        default: "SimpleInterval"
+        enum: [ "NumericInterval" ]
+        default: "NumericInterval"
       start:
-        type: ["integer"]
+        oneOf:
+          - $ref: "#/definitions/Numeric"
+          - $ref: "#/definitions/Range"
       end:
-        type: ["integer"]
-    example:
-      type: SimpleInterval
-      start: 11
-      end: 22
-
-  NestedInterval:
-    type: object
-    additionalProperties: false
-    properties:
-      type:
-        type: string
-        enum: ["NestedInterval"]
-        default: "NestedInterval"
-      inner:
-        $ref: '#/definitions/SimpleInterval'
-      outer:
-        $ref: '#/definitions/SimpleInterval'
-    example:
-      type: NestedInterval
-      inner:
-        type: SimpleInterval
-        start: 20
-        end: 30
-      outer:
-        type: SimpleInterval
-        start: 10
-        end: 40
+        oneOf:
+          - $ref: "#/definitions/Numeric"
+          - $ref: "#/definitions/Range"
+    # Require that Numerics (alone or in Ranges) are integers in NumericInterval
+    allOf:
+      - if:
+          properties:
+            start:
+              $ref: '#/definitions/Numeric'
+        then:
+          properties:
+            start:
+              properties:
+                value:
+                  type: integer
+                comparator:
+                  enum: [ "<", ">" ]
+        else:
+          properties:
+            start:
+              properties:
+                min:
+                  properties:
+                    value:
+                      type: integer
+                max:
+                  properties:
+                    value:
+                      type: integer
+      - if:
+          properties:
+            end:
+              $ref: '#/definitions/Numeric'
+        then:
+          properties:
+            end:
+              properties:
+                value:
+                  type: integer
+        else:
+          properties:
+            end:
+              properties:
+                min:
+                  properties:
+                    value:
+                      type: integer
+                max:
+                  properties:
+                    value:
+                      type: integer
+
+  # SimpleInterval has been moved to DEPRECATED section at bottom.
 
   CytobandInterval:
```

Comment:
Link this with previous comment about dropping Interval all together. If so, then CytobandInterval would go away and start/end HumanCytoband attributes would be moved directly onto the ChromosomeLocation concept.

Reply:
this is 2.0 dev